src/ott/tools/plot.py

```diff
@@ -224,3 +227,149 @@ def barycentric_projections(
     geom = arg.geom
 
   return _barycenters(ax, geom.y, arg.a, arg.b, arg.matrix, **kwargs)
+
+
+def plot_ot_map(
```

Comment:
Based on this [conversation](https://github.com/ott-jax/ott/pull/219#discussion_r1101940346), I thought these will become methods of `DualPotentials` and will move `potentials.py` from `ott/problems/linear/potentials.py` to `ott/tools/potentials.py`.

Reply:
I'm also with leaving the file in `ott/problems/linear/potentials.py`, the reason why it is `linear` is because we currently don't have notion of potentials for GW, pinging @marcocuturi again.

Reply:
I accidentally overlooked the part where you suggested to move all of `potentials.py` to tools, I thought @marcocuturi  just meant to move the plotting code into `ott/tools/plot`. I'm ok either way

Reply:
I strongly prefer this as a method of `DualPotentials`, makes more sense.

As for
> I noticed that ott.tools.plot isn't available in the docs because it's not listed [in tools.rst](https://github.com/ott-jax/ott/blob/main/docs/tools.rst). Do you want to add the docs for at least the new plotting code here or keep it disabled?

I'd keep it disabled, as I plan to refactor this in the future.

Reply:
@bamos could you please move the plotting functions as methods of `DualPotentials`?

Reply:
Added back there: https://github.com/ott-jax/ott/pull/219/commits/e53c33907492d259b9fd7b6f9a03ee6ed70bd3e6